source/Outfit.cpp

```diff
@@ -328,10 +332,18 @@ void Outfit::Load(const DataNode &node)
 					+ pluralName + "\".");
 	}
 
+	// Set the default jump fuel if not defined.
+	bool isHyperdrive = attributes.Get("hyperdrive"),
+		isScramDrive = attributes.Get("scram drive"),
+		isJumpDrive = attributes.Get("jump drive");
```

Comment:
```suggestion
	bool isHyperdrive = attributes.Get("hyperdrive");
	bool isScramDrive = attributes.Get("scram drive");
	bool isJumpDrive = attributes.Get("jump drive");
```
Might as well do this, as they are already in separate lines.